client/pages/timer-page.jsx

```diff
@@ -8,21 +8,37 @@ import Button from 'react-bootstrap/Button';
 import YellowSection from '../components/yellow-section';
 
 export default class TimerPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sessionTimes: []
+    };
+    this.addNewTime = this.addNewTime.bind(this);
+  }
+
+  addNewTime(time) {
+    const sessionTimes = this.state.sessionTimes.slice();
+    sessionTimes.push(time);
+    this.setState({
+      sessionTimes
+    });
+  }
+
   render() {
     return (
       <>
-        <Header></Header>
+        <Header />
         <Container>
           <Row>
             <Col lg className="mb-3">
               <Button className="std-button" block>Virtual Cube</Button>
             </Col>
             <Col lg>
-              <Timer></Timer>
+              <Timer addNewTime={this.addNewTime} />
             </Col>
           </Row>
         </Container>
-        <YellowSection></YellowSection>
+        <YellowSection contents="session" sessionTimes={this.state.sessionTimes} />
```

Comment:
It's unclear to me what this `contents` prop is doing besides adding indirection. If you want a container component with some pre-applied styling, I recommend implementing it to show its `props.children`

```jsx
<YellowSection>
  <SessionTimes sessionTimes={this.state.sessionTimes} />
</YellowSection>
```